base/i18n/base_i18n_export.h

```diff
@@ -4,8 +4,10 @@
 
 #ifndef BASE_I18N_BASE_I18N_EXPORT_H_
 #define BASE_I18N_BASE_I18N_EXPORT_H_
+#include "starboard/configuration.h"
 
-#if defined(COMPONENT_BUILD)
+// TODO b/296506338 Decide how to export symbols for base and net
+#if defined(COMPONENT_BUILD) || SB_IS(MODULAR) && !SB_IS(EVERGREEN)
```

Comment:
andrew mentioned that we disable component build gn flag intentionally to avoid building chrome components as shared libraries . MAybe @kaidokert  or someone else can speak to that .

Reply:
I've brought this up before as well that we should try and look at if we can lean on `COMPONENT_BUILD` from Chromium. However, this would constitute a fairly large re-think of how we are doing modular builds right now, and it comes with its own set of assumptions that don't necessarily hold in our build config.

I think pragmatically for now we just need to add our `SB_IS(MODULAR)` and make a follow-up ticket somewhere to take a look whether we could converge with `COMPONENT_BUILD` logic eventually. It's far bigger from Chrome side though, they build every major subcomponent as a DLL/dynlib and don't have the stable ABI concerns in between - they assume the same toolchain.